### PR TITLE
fix(editor): make the warning area update immediately

### PR DESCRIPTION
**Pull request by MrWindlike**

### CODE REVIEW

File: packages/editor/src/validator/rules/TraitRules.ts [hunk 1]
```diff
@@ -39,7 +39,8 @@ class EventHandlerValidatorRule implements TraitValidatorRule {
         });
       }
 
-      if (isExpression(targetId)) {
+      // TODO: util methods has no method schema to check the parameters, so now temporally skip validation
+      if (isExpression(targetId) || targetId === GLOBAL_UTILS_ID) {
```
Comment (Yuyz0112) [inline packages/editor/src/validator/rules/TraitRules.ts:43]: shall we validate the method without schema as `any`? Looks like currently, we allow users register method with a loose schema.

Comment (MrWindlike) [reply packages/editor/src/validator/rules/TraitRules.ts:43]: Yes. Editor may support auto complete the params of method, and it depends on the specific schema.

Comment (tanbowensg) [reply packages/editor/src/validator/rules/TraitRules.ts:43]: We may add spec for utilMethods in the future.

**Review by tanbowensg — APPROVED** [review]
We may add utilMethods spec in the future

**Pull request merged**